src/reducers/note.js

```diff
@@ -18,10 +19,12 @@ export const addNote = note => ({ type: NOTE_ADD, payload: note });
 export const updateCurrent = val => ({ type: CURRENT_UPDATE, payload: val });
 
 export const fetchNotes = () => async dispatch => {
+  dispatch(showMessage('Loading Notes'));
```

Comment:
@laurelherting calling dispatch on the showMessage action so it happens when fetchNotes() or saveNote() gets called from this note.js reducer